public/docs/content/.pages.yml

```diff
@@ -1,7 +1,8 @@
 ---
 nav:
-  - Welcome: welcome
+  - Overview: overview
+  - Getting started: getting_started
   - Developer Guides: developer_guides
-  - Walkthroughs: walkthroughs
   - API Reference: api
-  - FAQ: faqs
+  - Release Notes: release_notes
+  - Troubleshooting: faqs
```

Comment:
Lets remove this and move all the FAQ's directly under the package they pertain to.